src/js/api/global-api.js

```diff
@@ -33,7 +34,9 @@ define([
             return _addPlayer(new Api(domElement, _removePlayer));
         }
         // invalid query
-        return null;
+        return {
+            registerPlugin: plugins.registerPlugin
```

Comment:
We can move the legacy query behavior to commercial but it's a little weird:

jwplayer(invalid) returns {registerPlugin: function...}

vs

jwplayer(invalid) returns null

registerPlugin should go in the same place as registerProvider - currently that's in jwplayer.api, but jwplayer[here] makes more sense.